### PR TITLE
Add Touch Emoji

### PR DESCRIPTION
**Pull request by ilyalesik**
Add Touch Emoji

## Project URL

https://github.com/lm-a/touch-emoji

## Category

Utilities

## Description

Touch Emoji — emoji picker for MacBook Pro Touch Bar
 
## Checklist

- [x] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [x] Only one project/change is in this pull request
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

File: applications.json [hunk 1]
```diff
@@ -5442,6 +5442,19 @@
...
+      "title" : "Touch Emoji",
+      "screenshots" : [
+        "https://raw.githubusercontent.com/lm-a/touch-emoji/master/touch-emoji-icon-screenshot.png",
+        "https://raw.githubusercontent.com/lm-a/touch-emoji/master/touch-emoji-recent-screenshot.png"
+      ],
+      "short_description" : "Emoji picker for MacBook Pro Touch Bar. ",
+      "languages" : [
+        "swift"
+      ],
+      "category" : "utilities"
```
Comment (serhii-londar) [inline applications.json]: @ilyalesik I believe we need to create new category for such apps. For example 'touch_bar'? What do you think?

Comment (ilyalesik) [reply applications.json]: I think it's a good idea, although there are not many of them yet. 
MyTouchbarMyRules, Touch Bar Preview, Touch Bar Simulator can be in it.

Comment (ilyalesik) [reply applications.json]: @serhii-londar will you create the new category? Or should I just make changes to PR (categories.json).

Comment (serhii-londar) [reply applications.json]: Yes, please. You can do this in this PR.

Comment (ilyalesik) [reply applications.json]: Done it.

**Comment by serhii-londar** [comment]
@ilyalesik Thank you! :+1

**Comment by ilyalesik** [comment]
@serhii-londar `Touch Bar` category did not appear at Contents section. Was it should be added a manually to the README.md?

**Pull request merged**